build/GenerateCppJsRulesDescription.ps1

```diff
@@ -0,0 +1,65 @@
+param
+(
+    [Parameter(Mandatory = $true, HelpMessage = "target path to store the rules description")]
+    [ValidateScript({Test-Path $_})]
+    [string]
+    $targetFolder,
+
+    [Parameter(Mandatory = $true, HelpMessage = "path to rule extractor (from sonarlint-core). For example 'c:\src\sonarlint-rule-extractor-2.18-SNAPSHOT-jar-with-dependencies.jar' ")]
+    [ValidateScript({Test-Path $_})]
+    [string]
+    $ruleExtractor
+)
+
+function FindDaemonVersion($fileText)
```

Comment:
The convention I always followed is to use the Verb-Noun pattern for everything public, and pascal case for private functions. This is one of the reasons: https://stackoverflow.com/a/6032855/29430